codalab/apps/chahub/utils.py

```diff
@@ -29,6 +29,9 @@ def send_to_chahub(endpoint, data, update=False):
                 'X-CHAHUB-API-KEY': settings.CHAHUB_API_KEY,
             }
         }
-        return requests.post(**kwargs)
+        if update:
+            return requests.patch(**kwargs)
```

Comment:
chahub doesn't take patch requests, everything is done through post.

Reply:
cool will pull this